### PR TITLE
Added in get filenames

### PR DESCRIPTION
**Pull request by KFloodBCGov**
# Overview

This PR includes the following proposed change(s):

Added in the ability to get the bucket list, and tweaked the other endpooints:
Get File List:  GET http://localhost:8080/files
Get File:         GET http://localhost:8080/files/{filename}
Delete FIle:    DELETE http://localhost:8080/files/{filename}
Upload File:   POST http://localhost:8080/files (BODY contains multipart file)
-- see the POSTMAN file for more details

## Screenshots

Please add any relevant screenshots for your changes

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Locally using POSTMAN.

### CODE REVIEW

File: src/middleware/secure-file-storage-api/src/main/java/com/nttdata/securefilestorageapi/controller/FileController.java [hunk 1]
```diff
@@ -8,16 +8,17 @@
 import org.springframework.web.multipart.MultipartFile;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 
 @RestController
-@RequestMapping("/files")
+@CrossOrigin("http://localhost:3085")
```
Comment (sdevalapurkar) [inline src/middleware/secure-file-storage-api/src/main/java/com/nttdata/securefilestorageapi/controller/FileController.java:15]: can we make this an env var so each person can set it themselves? for ex, my fe is running on 3000 not 3085

Comment (KFloodBCGov) [reply src/middleware/secure-file-storage-api/src/main/java/com/nttdata/securefilestorageapi/controller/FileController.java]: I think I got that in?  Let me know if it isn't working...

File: src/middleware/secure-file-storage-api/src/main/java/com/nttdata/securefilestorageapi/controller/FileController.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 import java.util.Map;
 
 @RestController
-@CrossOrigin("http://localhost:3085")
+@CrossOrigin(origins = {"${allowed.origins}"})
```
Comment (sdevalapurkar) [inline src/middleware/secure-file-storage-api/src/main/java/com/nttdata/securefilestorageapi/controller/FileController.java:15]: nice

**Review by sdevalapurkar — APPROVED** [review]
lgtm

**Pull request merged**